datajunction-server/datajunction_server/models/materialization.py

```diff
@@ -142,6 +142,31 @@ class MaterializationConfigInfoUnified(
     """
 
 
+class AvailabilityStateInfo(BaseModel):
+    """
+    Availability state information for a node
+    """
+
+    id: int
+    catalog: str
+    schema_: Optional[str] = Field(alias="schema")
+    table: str
+    valid_through_ts: int
+    url: Optional[str] = None
+    links: Optional[Dict] = None
+    categorical_partitions: Optional[List[str]] = []
```

Comment:
btw, assigning this to `[]` (which is a mutable object) could cause unexpected issues, because `AvailabilityStateInfo` will share the same list object for this field (and for each of the fields below) unless overridden. 

For pydantic models you could default to this instead: `Field(default_factory=list)`